### ISSUE TITLE
did any body know how to use this code only train rpn network?

### ISSUE THREAD

**Issue by mlinxiang** [jwyang/faster-rcnn.pytorch#347]

**Comment by jwyang** [comment]
@mlinxiang mute the loss term for bbox classification. Then, set the hyper parameters according to the tables in readme.

**Comment by xueyushan** [comment]
@jwyang How to mute the loss term for bbox classification?
loss = rpn_loss_cls.mean() + rpn_loss_box.mean()+ 0.0*RCNN_loss_cls.mean() + 0.0*RCNN_loss_bbox.mean()   is right?
